spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -432,239 +425,252 @@ dataflow:>task list
+
+`task create my-split-task --definition "<foo || bar || baz>"`
+
+The preceding example above launches tasks `foo`, `bar` and `baz` in parallel.
+
+Using the Spring Cloud Data Flow Dashboard to create the same "`split execution`" would resemble the following image:
 
 .Split
 image::{dataflow-asciidoc}/images/dataflow-ctr-split.png[Composed Task Split, scaledwidth="50%"]
 
-With the task DSL a user may also execute multiple split groups
-in succession. For example:
-```
-task create my-split-task --definition "<foo || bar || baz> && <qux || quux>"
-```
+With the task DSL, a user may also execute multiple split groups in succession, as shown in the following example:
 
-In the example above tasks `foo`, `bar` and `baz` will be launched in parallel,
-once they all complete then tasks `qux`, `quux` will be launched in parallel.
-Once they complete the composed task will end.   However if `foo`, `bar`, or
-`baz` fails then, the split containing `qux` and `quux` will not launch.
+`task create my-split-task --definition "<foo || bar || baz> && <qux || quux>"'
 
-Using the Spring Cloud Data Flow Dashboard to create the same
-"split with multiple groups" would look like:
+In the preceding example, tasks `foo`, `bar`, and `baz` are launched in parallel.
+Once they all complete, then tasks `qux` and `quux` are launched in parallel.
+Once they complete, the composed task ends.
+However, if `foo`, `bar`, or `baz` fails, the split containing `qux` and `quux` does not launch.
+
+Using the Spring Cloud Data Flow Dashboard to create the same "`split with multiple groups`" would resemble the following image:
 
 .Split as a part of a conditional execution
 image::{dataflow-asciidoc}/images/dataflow-ctr-multiple-splits.png[Composed Task Split, scaledwidth="50%"]
 
-Notice that there is a `SYNC` control node that is by the designer when
+Notice that there is a `SYNC` control node that is inserted by the designer when
 connecting two consecutive splits.
 
+
+
 ==== Split Containing Conditional Execution
-A split can also have a conditional execution within the angle brackets.  For
-example:
-```
-task create my-split-task --definition "<foo && bar || baz>"
-```
-In the example above we see that `foo` and `baz` will be launched in parallel,
-however `bar` will not launch until `foo` completes successfully.
 
-Using the Spring Cloud Data Flow Dashboard to create the same
-"split containing conditional execution" would look like:
+A split can also have a conditional execution within the angle brackets, as shown in the following example:
+
+`task create my-split-task --definition "<foo && bar || baz>"`
+
+In the preceding example, we see that `foo` and `baz` are launched in parallel.
+However, `bar` does not launch until `foo` completes successfully.
+
+Using the Spring Cloud Data Flow Dashboard to create the same "`split containing conditional execution`" resembles the following image:
```

Comment:
Need to add a blank between the " and the single tick, else the formatting does not work.